ern-core/src/iosUtil.ts

```diff
@@ -578,7 +578,9 @@ export const getDefaultIosDeploymentTarget = (rnVersion: string): string => {
 export const getReactNativeCodegenVersion = (
   rnVersion: string,
 ): string | null => {
-  if (semver.gte(rnVersion, '0.67.0')) {
+  if (semver.gte(rnVersion, '0.68.4')) {
+    return '^0.0.18';
+  } else if (semver.gte(rnVersion, '0.67.0')) {
```

Comment:
For completeness, can you also add the following cases:

gte `0.68.2` => codegen `^0.0.17`
gte `0.68.0` => codegen `^0.0.13`

Reply:
sure!